cmd/system-probe/modules/ebpf.go

```diff
@@ -58,6 +59,17 @@ func (o *ebpfModule) Register(httpMux *module.Router) error {
 		utils.WriteAsJSON(w, stats)
 	}))
 
+	httpMux.HandleFunc("/btf_loader_info", func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
This module is actually for the ebpf check and isn't always enabled

Reply:
For customers it will basically never be enabled, because it isn't intended to be a customer-facing feature.

Reply:
Understood- just so I can learn more, what is the ebpf check? (Is there a central documentation of this kind of thing?)

Reply:
Also, where do you think I should move it to? Originally it was in the `NetworkTracer` module, but BTF information is used by more than just networks. One option is to put it at the top-level routes but I wasn't sure if that was desirable either.

Reply:
> what is the ebpf check?

The ebpf check gathers metrics about ebpf usage on the host. Map memory usage, map entry counts, program size, etc.

Reply:
> One option is to put it at the top-level routes

I think this makes the most sense.

Reply:
Moved it to the top level routes